guides/using_flagging.md

```diff
@@ -3,31 +3,35 @@
 related_spaces: https://huggingface.co/spaces/aliabd/calculator-flagging-crowdsourced, https://huggingface.co/spaces/aliabd/calculator-flagging-options, https://huggingface.co/spaces/aliabd/calculator-flag-basic
 tags: FLAGGING, DATA
 
-## The `flag` button 
+## Introduction
 
-Underneath the output interfaces, there is a button marked `flag`. When a user testing your model sees input with interesting output, such as erroneous or unexpected model behaviour, they can flag the input for the interface creator to review. 
+When you deploy or demo a machine learning model, you may find that it behaves differently than how you expected (e.g. the model makes an incorrect prediction) when a user tries it with their own data. Capturing these "hard" data points is important because it allows you to make you machine learning model more reliable and robust.
+
+Gradio simplifies the collection of this data by including a FLAG button with every `Interface`. This allows your user or tester to easily send data back to you, whether the model is running locally or has been shared by setting `share=True`. 
+
+## The **Flag** button 
+
+Underneath the output interfaces, there is a button marked **Flag**. When a user testing your model sees input with interesting output, such as erroneous or unexpected model behaviour, they can flag the input for the interface creator to review. 
 
 ![flag button](website/src/assets/img/guides/using_flagging/flag_button.gif)
 
 There are four parameters `gr.Interface` that control how flagging works. We will go over them in greater detail.
 
-* `allow_flagging`: 
-    * This parameter can be set to either `"manual"`, `"auto"`, or `"never"`.                 
+* `allow_flagging`: this parameter can be set to either `"manual"`, `"auto"`, or `"never"`.                 
     * `manual`: users will see a button to flag, and events are only flagged when it's clicked.
     * `auto`: users will not see a button to flag, but every event will be flagged automatically. 
     * `never`: users will not see a button to flag, and no event will be flagged. 
-* `flagging_options`:
-    * This parameter takes a list of strings.
+* `flagging_options`: this *optional* parameter takes a list of strings.
```

Comment:
what does optional mean here? all of 4 of these parameters are optional

Reply:
As in, it can be set to None. We should actually add that if it is set to None, then no options will be provided, and flagging will be more of a binary thing. Let me update it.

Reply:
Hmm, not sure that's relevant, since its None by default. Why would a user want to set it to None?

Reply:
We still need to mention that it can be either `None` or a list of strings. Otherwise, the specification for the parameter is incomplete. Take a look at how I've written it now, I think it's pretty clear?